packages/launchpad/cypress/e2e/project-setup.cy.ts

```diff
@@ -259,8 +259,6 @@ describe('Launchpad: Setup Project', () => {
         cy.findByRole('button', { name: 'Continue' })
         .should('not.have.disabled')
         .click()
-
-        verifyChooseABrowserPage()
```

Comment:
covered in many other places - covering it here would require us to add `dependencies` to the `pristine` project, which would slow down the run for no additional coverage